public/API/API_GetClaims.php

```diff
@@ -32,6 +28,11 @@
  *    int        MinutesLeft        time in minutes left until the claim expires
  */
 
+use App\Community\Enums\ClaimFilters;
+use Illuminate\Support\Arr;
```

Comment:
I don't think this is needed.

Reply:
If it's removed, my IDE starts throwing a red squiggly underneath `Arr`:
![Screenshot 2025-05-21 at 6 13 25 PM](https://github.com/user-attachments/assets/35ca0bab-3da6-439d-8a68-301b2caeccd3)